<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYZTextEditor</class>
 <widget class="QWidget" name="XYZTextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QToolButton" name="btnCut">
       <property name="toolTip">
        <string>Cut</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/cut.png</normaloff>:/new/prefix1/images/cut.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCopy">
       <property name="toolTip">
        <string>Copy</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/copy.png</normaloff>:/new/prefix1/images/copy.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPaste">
       <property name="toolTip">
        <string>Paste</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/paste.png</normaloff>:/new/prefix1/images/paste.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnTextBold">
       <property name="toolTip">
        <string>Bold</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/textbold.png</normaloff>:/new/prefix1/images/textbold.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnTextItalic">
       <property name="toolTip">
        <string>Italic</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/textitalic.png</normaloff>:/new/prefix1/images/textitalic.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnUnderline">
       <property name="toolTip">
        <string>Underline</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/textunder.png</normaloff>:/new/prefix1/images/textunder.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnAlignLeft">
       <property name="toolTip">
        <string>Align left</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/textleft.png</normaloff>:/new/prefix1/images/textleft.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnAlignCenter">
       <property name="toolTip">
        <string>Align center</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/textcenter.png</normaloff>:/new/prefix1/images/textcenter.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnAlignRight">
       <property name="toolTip">
        <string>Align right</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/textright.png</normaloff>:/new/prefix1/images/textright.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnAlignJustify">
       <property name="toolTip">
        <string>Align jusify</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/textjustify.png</normaloff>:/new/prefix1/images/textjustify.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboStyle">
       <property name="toolTip">
        <string>List</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnTextColor">
       <property name="toolTip">
        <string>Font color</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtRptDesigner/images.qrc">
         <normaloff>:/new/prefix1/images/fontColor.png</normaloff>:/new/prefix1/images/fontColor.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFontComboBox" name="comboFont">
       <property name="toolTip">
        <string>Font</string>
       </property>
       <property name="currentFont">
        <font/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboSize">
       <property name="toolTip">
        <string>Font size</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string notr="true">6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">26</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">28</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">36</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">48</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">72</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnTextDirection">
       <property name="toolTip">
        <string>TextDirection</string>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtRptDesigner/images.qrc"/>
 </resources>
 <connections/>
</ui>
